AVFoundation-CombineTests/TestSubscriber.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  TestSubscriber.swift
+//  AVFoundation-CombineTests
+//
+//  Created by József Vesza on 2020. 07. 24..
+//  Copyright © 2020. József Vesza. All rights reserved.
+//
+
+import Combine
+
+class TestSubscriber<T>: Subscriber {
+    typealias Input = T
+    typealias Failure = Never
+    
+    let demand: Int
+    let onComplete: ([T]) -> Void
+    
+    private var receivedValues: [T] = []
+    private var subscription: Subscription? = nil
+    
+    init(demand: Int, onComplete: @escaping ([T]) -> Void) {
```

Comment:
@jozsef-vesza as in my previous comment, the demand could be of type `Demand` so it can be set as **unlimited**